scripts/RetentionScript/RetentionScript.ps1

```diff
@@ -0,0 +1,194 @@
+<#
+ .SYNOPSIS
+    This powershell module deletes backups from the azure storage.
```

Comment:
Reword: This powersehll script helps delete older backups from Azure storage or file share which were taken using the Service Fabric Backup Restore service.